src/Drupal/Commands/core/LocaleCommands.php

```diff
@@ -155,6 +159,64 @@ public function update($options = ['langcodes' => self::REQ])
         drush_backend_batch_process();
     }
 
+    /**
+     * Exports to a gettext translation file.
+     *
+     * See Drupal Core: \Drupal\locale\Form\ExportForm::submitForm
+     *
+     * @throws \Exception
+     *
+     * @command locale:export
+     * @param $langcode The language code of the exported translations.
+     * @option template POT file output of extracted source texts to be translated.
+     * @option types String types to include, defaults to all types. Types: 'not-customized', 'customized', 'not-translated'.
+     * @usage drush locale:export nl > nl.po
+     *   Export the Dutch translations with all types.
+     * @usage drush locale:export nl --types=customized,not-customized > nl.po
+     *   Export the Dutch customized and not customized translations.
+     * @usage drush locale:export --template > drupal.pot
+     *   Export the basic template file to translate.
+     * @usage drush locale:export --template nl > nl.pot
+     *   Export the Dutch template file to translate.
+     * @aliases locale-export
+     * @validate-module-enabled locale
+     */
+    public function export($langcode = null, $options = ['template' => false, 'types' => self::OPT])
+    {
+        $language = $this->getTranslatableLanguage($langcode);
+        $poreader_options = [];
+
+        if (!(bool)$options['template']) {
```

Comment:
The default value of --template is `false` which tells AnnotatedCommand that no string values are allowed, only bool. We aren't great anymore with options that can be bool or string. I think its possible, but awkward.

Reply:
'template' is not used as string. I think the explicit type casting causes the confusion. The casting is not needed, now removed.